analyzers/tests/SonarAnalyzer.UnitTest/Facade/SyntaxFacadeTest.cs

```diff
@@ -82,12 +82,12 @@ public class SyntaxFacadeTest
             vb.NodeIdentifier(null).Should().BeNull();
 
         [TestMethod]
-        public void NodeIdentifier_UnexpectedTypeThrows_CS() =>
-            cs.Invoking(x => x.NodeIdentifier(CS.SyntaxFactory.ThrowStatement())).Should().Throw<InvalidOperationException>();
+        public void NodeIdentifier_Unexpected_Returns_Null_CS() =>
```

Comment:
Also `InvocationIdentifier_UnexpectedTypeThrows_CS` and `InvocationIdentifier_UnexpectedTypeThrows_VB` needs to be updated after the change of exception type